docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport Auth
+
+{WFM-RC-NameShort} authentication and authorization module using link:http://passportjs.org[PassportJS].
```

Comment:
I see this being copy paste from actual repo. While the content is ok I think that product documentation should lot different. Let's avoid pasting large pieces of the code (this can be found as well in module README anyway)

When describing module we may look for some typical patterns like:
- why this module is being introduced
- When user will need to use it 
- What are the extension points (implementation required from developers - if any) etc.

Module README's may have some useful information for this but they almost every time will require additional formating etc.